notebooks/create-validation-dataset.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        if line.startswith('###'):
</pre><p> </p><p>A minor thing: we also want to skip lines that start with <code>## </code>  - the headings for the categories/groups. The current code seems to get these included as part of some answers to previous questions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>